src/server/Document/component.jsx

```diff
@@ -66,6 +68,7 @@ const Document = ({
         {isAmp && (
           <>
             {AMP_JS}
+            {hasAds && AMP_ADS_JS}
```

Comment:
Hey! I just had a quick look at this draft PR - it's probably going to easier to add the AMP_ADS script that's already been added psammead using react helmet in the AMP Ads component, that way it will be appended to the document for all pages that are using ads automatically :) - we do the same here for the AMP_ACCESS_JS script: https://github.com/bbc/simorgh/blob/064769bd5cac2e9d30e48df2334351039a52eb20/src/app/containers/Ad/Amp/index.jsx#L86

Reply:
Thanks for taking a look. I chose this approach because it was my understanding that the reason is that the script needs to be added into the Document in a particular order (AMP_AD_JS needs to be after the standard AMP_JS) in that file. So we can’t add it in /containers/Ad/Amp.

Reply:
Aha. OK, so Helmet puts the scripts that we add like this into components above the AMP_JS script, due to the order we list them in the Document. I'm not sure that it's a problem that we add the `AMP_ADS_JS` script tag before the `AMP_JS` script tag, as they are both loaded asynchronously (I'll check this). We appear to already load `amp-bind` before the main AMP script on live pages. If we are concerned about this, we could move the AMP_JS script in the Document above the scripts appended by Helmet, rather than needing to add more logic? Maybe worth a quick chat in/after stand up :)

Reply:
Adding AMP_ADS_JS to helmet in the container as suggested fixes the warning shown by amp validator, without having to reposition AMP_JS script in the Document

Reply:
Sounds like a good first step, we can see if we need to do anything more complicated afterwards :)